File: sub-model/src/main/java/org/github/base/service/impl/SysCodeService.java
package org.github.base.service.impl;

import org.github.base.entity.SysCodeEntity;
import org.github.base.mapper.SysCodeMapper;
import org.github.base.service.ISysCodeService;
import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * sys_code_table 服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2019-01-06
 */
@Service
public class SysCodeService extends ServiceImpl<SysCodeMapper, SysCodeEntity> implements ISysCodeService {}

File: sub-mysql-sccore-service/src/main/java/org/github/mysql/sccore/base/service/impl/SysCodeService.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.sccore.base.service.impl;

import org.github.mysql.sccore.base.entity.SysCodeEntity;
import org.github.mysql.sccore.base.mapper.ISysCodeMapper;
import org.github.mysql.sccore.base.service.ISysCodeService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * 数据字典表 服务实现类
 *
 * @author JYD_XL
 */
@Service
open class SysCodeService : ServiceImpl<ISysCodeMapper, SysCodeEntity>(), ISysCodeService
